packages/devtools/lib/src/memory/memory_chart.dart

```diff
@@ -18,8 +18,7 @@ class MemoryChart extends CoreElement {
     element.id = _memoryGraph;
     element.style
       ..boxSizing = 'content-box' // border-box causes right/left border cut.
-      ..height = '${FramesBarChart.chartHeight}px'
-      ..width = '100%';
```

Comment:
Quick question was 100% causing a problem or just unnecessary?

Reply:
width = '100%' was causing the right border of the graph to be cut off.  The chart still takes up the full width without this line, and the border is now visible 👍

Reply:
Great! Thx.